### PR TITLE
Update 01-08

### LINKED ISSUE #7: 🚀 [Feature] Person Ulf Gebhardt

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

Update from the 08.01.2023 based on #15 

image

### Issues

- fixes #7 

### Stuff without ticket
- configure `@images` alias

### Todo

- [x] Link projects internally
- [x] Link tech
- [x] Jobs in Norway
- [x] Travel Halden -> Gotenborg, Hamburg -> Berlin
- [x] Separate Travels

### CODE REVIEW

File: docs/people/ulf-gebhardt.md [hunk 1]
```diff
@@ -1,5 +1,140 @@
...
+
+## Role
+
+Ulf can fulfill the following roles:
+
+- DevOps
+- Scrum Master
+- Softwarearchitekt
+- Backend Developer
+- Strategist & Planner
```
Comment (Elweyn) [inline docs/people/ulf-gebhardt.md:19]: I would have named it "Project Manager" or "Project Management"

File: docs/people/ulf-gebhardt.md [hunk 2]
```diff
@@ -1,5 +1,140 @@
...
+- Database planning and optimization
+- Competence assessment
+- SEO
+
+## Spoken languages
+
+Ulf speaks German and English fluently.
+
+- German (mother tongue)
+- English (orally and in writing)
```
Comment (Elweyn) [inline docs/people/ulf-gebhardt.md:30]: "Orally" sounds strange for me! "Speak and write"! Oder "level C1"!

File: docs/people/ulf-gebhardt.md (hunk 1 above)
Comment (ulfgebhardt) [reply docs/people/ulf-gebhardt.md]: Fixed in https://github.com/IT4Change/IT4C.dev/pull/20/commits/92bd43499b4bb6f21338df26d9a71bb89952edda

File: docs/people/ulf-gebhardt.md (hunk 2 above)
Comment (ulfgebhardt) [reply docs/people/ulf-gebhardt.md:30]: For me that is standard English, see https://www.linguee.de/deutsch-englisch/search?source=auto&query=orally+and+in+writing

Comment (Elweyn) [reply docs/people/ulf-gebhardt.md:30]: OK sound just strange for me!

**Review by Elweyn — APPROVED** [review]

**Pull request merged**